pkg/dao/repository_config_test.go

```diff
@@ -267,7 +267,7 @@ func (suite *RepositoryConfigSuite) TestBulkCreateOneFails() {
 
 func (suite *RepositoryConfigSuite) TestUpdate() {
 	name := "Updated"
-	url := "http://someUrl.com"
+	url := "http://someUrl.com/"
```

Comment:
as above

Reply:
this is actually part of https://github.com/content-services/content-sources-backend/pull/121  that i included here